src/CMakeLists.txt

```diff
@@ -777,3 +777,5 @@ endif() # NOT WIN32
 if ( NOT WIN32)
   add_subdirectory(test test)
 endif() # NOT WIN32
+
+add_subdirectory(python python)
```

Comment:
When running CI pipelines. make sure this PR doesn't break windows build, cpu-only builds, `L0_build_variants`, etc